contracts/loadtester/LoadTester.go

```diff
@@ -1,7 +1,7 @@
 // Code generated - DO NOT EDIT.
 // This file is a generated binding and any manual changes will be lost.
 
-package loadtester
+package contracts
```

Comment:
Is this right?

Reply:
this one is a weird one. It's an unused file and the one used instead is `contracts/loadtester.go`. I deleted this one altogether. 

Also tested out `mode p` to make sure it works and everything still seems functional:
```
$ go run main.go loadtest \
                --legacy \
                --verbosity 700 \
                --mode p \
                --rate-limit 10 \
                --requests 10 \
                --concurrency 10
```